packages/modules/src/types.ts

```diff
@@ -0,0 +1,5 @@
+import { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+
+export type ReactComponent<GenericProps = Record<string, never>> = (
+    props: GenericProps,
+) => EmotionJSX.Element;
```

Comment:
❤️

Reply:
is it worth adding a comment about the reason for this?

Reply:
👍  Great idea, I've done this.